monkey/agent_plugins/exploiters/hadoop/config-schema.json

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Hadoop/Yarn Exploiter",
+    "description": "Remote code execution on HADOOP server with YARN and default settings. Logic based on https://github.com/vulhub/vulhub/tree/master/hadoop/unauthorized-yarn.",
+    "required": [
+        "exploitable_ports",
+        "http_download_timeout",
+        "app_random_string_length"
+    ],
```

Comment:
`id_string` should be required too. We need it for the random name for the application.